packages/bits/src/lib/expander/expander.component.html

```diff
@@ -1,6 +1,7 @@
 <div class="nui-expander"
      [class.nui-expander--expanded]="state === 'expanded'">
     <div class="nui-expander__header"
+         role="button"
```

Comment:
`switch` might be even better since this button is a toggle.

Reply:
agree. Will add, thanks